packages/docs/docs/guides/5-request-client/1-payment-networks.md

```diff
@@ -5,42 +5,54 @@ keywords: Request Client, Payment networks
 
-When a user creates and sends a request, he can expect to receive the correct amount of money. But how does he keep track of the payments due and received?
+There are different methods available for the requester and payer to agree on the payment status, and that is when payment networks come into play. A payment network is a specific context on how to agree on the payment status of a specific request.
 
-There are different methods available for the requester and payer to agree on the payment status, and that is when Payment networks come into play.
+A payment network is defined by:
+* An extension in the Request protocol that defines the necessary information to be able to detect payments
+* A payment method, it's the method used to perform a detectable payment
+* A payment detection method, it's the process to determine the amount paid by the payer through the payment method
 
-Payment networks allow developers to:
-* Detect blockchain-based payment automatically
-* Declare other kinds of payments completion
+For example, the Address-Based Bitcoin payment network, `pn-bitcoin-address-based`, allowing to detect Bitcoin payments through unique Bitcoin addresses created for the request's payments and refunds. For this payment network:
+* The Request protocol's extension stores the Bitcoin payment and refund addresses of the request
+* The payment method is to perform a simple Bitcoin transfer to the payment or the refund address
+* The payment detection method is to read the inbound BTC transfers into the payment Bitcoin address to determine the paid amount and to read the inbound BTC transfers into the refund Bitcoin address to determine refunded amount
 
-Request is blockchain agnostic, payment methods that are suppored for automated payment dection are: Bitcoin, Ether and ERC20. If you miss your favorite blockchain, you can either get in touch with the team or **TODO learn how to do it yourself**
+Note that these characteristics are almost the same for every address-based payment network.
 
-## Ethereum and ERC20
+For most payment networks, the payment method can be performed by the payer without any interaction with the Request protocol or the Request client.
+For all payment networks, the Request client can perform the payment detection method to determine the balance of a request.
 
-Because one Ethereum address is generally used many times to receive and send transactions, we need a way to identify each payment. Request provides two payment channels for ETH and ERC20:
+Currencies that are supported for automated payment detection are Bitcoin, Ether, and ERC20. If you miss your favorite blockchain, you can either get in touch with the team or **TODO learn how to do it yourself**
```

Comment:
"Currencies that are supported for automated payment detection are Bitcoin, Ether, and ERC20"
It would be more clear if written as a list so. It will make it easier to spot that a new currency have been added.